code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -87,6 +89,126 @@
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/orange(location)
+
+/datum/chemical_reaction/crayonicationyellow
+	name = "Yellow Crayon"
+	id = "yellowcrayonication"
+	required_reagents = list("lye" = 10, "yellowcrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+
+/datum/chemical_reaction/crayonicationyellow/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/yellow(location)
+
+/datum/chemical_reaction/crayonicationgreen
+	name = "Green Crayon"
+	id = "greencrayonication"
+	required_reagents = list("lye" = 10, "greencrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+
+/datum/chemical_reaction/crayonicationgreen/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/green(location)
+
+/datum/chemical_reaction/crayonicationblue
+	name = "Blue Crayon"
+	id = "bluecrayonication"
+	required_reagents = list("lye" = 10, "bluecrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+
+/datum/chemical_reaction/crayonicationblue/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/blue(location)
+
+/datum/chemical_reaction/crayonicationpurple
+	name = "Purple Crayon"
+	id = "purplecrayonication"
+	required_reagents = list("lye" = 10, "purplecrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+
+/datum/chemical_reaction/crayonicationpurple/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/purple(location)
+
+/datum/chemical_reaction/crayonicationblack
+	name = "Black Crayon"
+	id = "blackcrayonication"
+	required_reagents = list("lye" = 10, "blackcrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+
+/datum/chemical_reaction/crayonicationblack/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/black(location)
+
+/datum/chemical_reaction/crayonicationwhite
+	name = "White Crayon"
+	id = "whitecrayonication"
+	required_reagents = list("lye" = 10, "whitecrayonpowder" = 10)
+	required_temp = 374
+	mob_react = FALSE
+
+/datum/chemical_reaction/crayonicationwhite/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
+		new /obj/item/toy/crayon/white(location)
+
+/datum/chemical_reaction/crayonicationmime
+	name = "Mime Crayon"
+	id = "mimecrayonitation"
+	required_reagents = list("lye" = 10, "whitecrayonpowder" = 10, "blackcrayonpowder" =10)
```

Comment:
the black or white reaction will happen before this, making this impossible to achieve; maybe make it 5 and 5

Reply:
replace lye with nothing